modules/nw-configuring-qinq-sriov-proc.adoc

```diff
@@ -0,0 +1,243 @@
+spec:
+  deviceType: netdevice
+  nicSelector:
+    pfNames:
+      - ens5f0#0-9
+  nodeSelector:
+    node-role.kubernetes.io/worker-cnf: ""
+  numVfs: 10
+  priority: 99
+  resourceName: resource810
+----
+
+. Optional: Label the SR-IOV capable cluster nodes with `SriovNetworkNodePolicy.Spec.NodeSelector` if they are not already labeled. For more information about labeling nodes, see "Understanding how to update labels on nodes".
+
+. Create the `SriovNetworkNodePolicy` object:
++
+[source,terminal]
+----
+$ oc create -f <name>-sriov-node-network.yaml
+----
++
+After applying the configuration update, all the pods in `sriov-network-operator` namespace move to the `Running` status.
+
+. To verify that the SR-IOV network device is configured, enter the following command. Replace <node_name> with the name of a node with the SR-IOV network device that you just configured.
++
+[source,terminal]
+----
+$ oc get sriovnetworknodestates -n openshift-sriov-network-operator <node_name> -o jsonpath='{.status.syncStatus}'
+----
+
+. Create a `SriovNetwork` object, setting the outer VLAN referred to as the S-tag, or `Service Tag`, as it belongs to the infrastructure. Save the YAML in the <name>.yaml file, where <name> is a name for this additional network. The object specification might resemble the following example:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriovnetwork-1ad-810
+  namespace: openshift-sriov-network-operator
+spec:
+  ipam: '{}'
+  vlan: 171 <1>
+  vlanProto: "802.1ad" <2>
+  networkNamespace: default
+  resourceName: resource810
+----
++
+<1> This sets the S-tag VLAN tag to 171. 
+<2> Specifies the VLAN protocol to assign to the VF. Allowed values are `802.1ad` and `802.1q`. The default is `802.1q`. 
+
+. To create the object, enter the following command:
++
+[source,terminal]
+----
+$ oc create -f <name>.yaml
+----
+
+. Optional: To confirm that the `NetworkAttachmentDefinition` object that is associated with the `SriovNetwork` object that you created in the previous step exists, enter the following command. Replace `<namespace>` with the networkNamespace you specified in the `SriovNetwork` object.
++
+[source,terminal]
+----
+$ oc get net-attach-def -n default
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                   AGE
+sriovnetwork-1ad-810   29s
+----
+
+. Create a Network Attachment Definition with an inner VLAN. The inner VLAN is often referred to as the C-tag, or `Customer Tag`, as it belongs to the Network Function.
+
+.. Using the following YAML example, create a file named `nad-cvlan100.yaml`:
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: nad-cvlan100
+  namespace: default
+spec:
+  config: |
+    {
+      "cniVersion": "0.3.1",
+      "name": "vlan-100",
+      "plugins": [
+        {
+          "type": "vlan",
+          "master": "net1", 
```

Comment:
you need to mentoin that the master corresponds to the VF interface inside the pod. Default name is net1 since name is not set in the pod annotion